src/ray/raylet_client/raylet_client.h

```diff
@@ -20,206 +20,18 @@
 #include <unordered_map>
 #include <vector>
 
-#include "ray/common/asio/instrumented_io_context.h"
-#include "ray/common/buffer.h"
-#include "ray/common/bundle_spec.h"
-#include "ray/common/status.h"
-#include "ray/common/status_or.h"
-#include "ray/ipc/client_connection.h"
-#include "ray/rpc/node_manager/node_manager_client.h"
-#include "ray/util/process.h"
-#include "src/ray/protobuf/common.pb.h"
-#include "src/ray/protobuf/gcs.pb.h"
-
-using ray::ActorID;
-using ray::JobID;
-using ray::NodeID;
-using ray::ObjectID;
-using ray::TaskID;
-using ray::WorkerID;
-
-using ray::Language;
+#include "ray/raylet_client/raylet_client_interface.h"
 
 // Maps from resource name to its allocation.
 using ResourceMappingType =
     std::unordered_map<std::string, std::vector<std::pair<int64_t, double>>>;
 
```

Comment:
Should we consider moving the alias `ResourceMappingType` out of the interface header into implementation or a dedicated types header if it causes extra includes in dependents?

Reply:
I think it is already in the implementation header. We don't have `ResourceMappingType` in the `raylet_client_interface.h`.